dimagi/pages/templates/amp/base.amp.html

```diff
@@ -0,0 +1,247 @@
+{% load i18n %}{% load dimagidotcom %}{% load capture_tags %}<!DOCTYPE html>
+{# note: modernizr.js and app.js add classes to <html> to trigger certain css behaviors. #}
+{# These classes are: #}
+{#     js, no-js, touch, no-touch, cssanimations, #}
+{#     csstransformations3d, csstransitions, video, audio, svg, inlinesvg #}
+{#     pointerevents, show-time #}
+<html amp lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="width=device-width,minimum-scale=1, initial-scale=1, user-scalable=no, shrink-to-fit=no, viewport-fit=cover">
+    <meta name="format-detection" content="telephone=no">
```

Comment:
if these meta tags are to be shared between the two base templates, then they should be pulled into a partial so that we aren't changing the same meta tag in two different places.

Reply:
(same comment as below):
better to keep the same `base.html` template and use a server-side AMP flag determined by the User Agent to hide / show the amp content changes on the original `base.html` rather than creating a whole separate base template, as there is too much duplicate code otherwise.